Dockerfile-base

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/mozmar/base:v0.3.5
+FROM quay.io/mozmar/base:v0.3.6
```

Comment:
If you remove the tag and use the implicit `latest`, you won't have to file a PR like this again. That's actually the main reason we have that base.